tests/unit/Iterator/PsrCacheIteratorTest.php

```diff
@@ -69,20 +73,12 @@ public function it_can_cache_data(CacheItemPoolInterface $cache): void
             ->hasItem(1)
             ->willReturn(false);
 
-        $this->next();
+        $iterator->next();
 
-        $this
-            ->current()
-            ->shouldReturn('b');
+        self::assertEquals('b', $iterator->current());
```

Comment:
```suggestion
        self::assertSame('b', $iterator->current());
```

this applies everywhere

Reply:
I replaced `assertEquals` with `assertSame` everywhere.